build.ps1

```diff
@@ -10,9 +10,10 @@ Write-Host "  CreatePackages: $CreatePackages"
 Write-Host "  RunTests: $RunTests"
 Write-Host "  dotnet --version:" (dotnet --version)
 
-$packageOutputFolder = "$PSScriptRoot\.nupkgs"
+$packageOutputFolder = Join-Path $PSScriptRoot ".nupkgs"
 $projectsToBuild =
-    'StackExchange.Utils.Http'
+    'StackExchange.Utils.Http',
+    'StackExchange.Utils.Configuration'
```

Comment:
We should probably use the Build.csproj version of this now - see https://github.com/StackExchange/StackExchange.Redis/blob/main/build.ps1 - means we no longer list the projects at all!